src/main/java/donnafin/logic/commands/AppendCommand.java

```diff
@@ -54,11 +54,11 @@
     public static final String MESSAGE_DUPLICATE_POLICY = "Sorry, the policy with this exact specifications"
```

Comment:
it should be 'a policy with...'. can do the same for assets and liability?

Reply:
oki